.github/workflows/release.yml

```diff
@@ -1,28 +1,33 @@
 ---
-name: release
+name: Release
 
 permissions:
   contents: read
 
 on:
   workflow_dispatch:
     inputs:
-      branch_specifier:
-        description: The branch to release ex. main or 0.6.
+      ref:
+        description: 'Branch or tag ref to run the workflow on'
         required: true
         default: "main"
-        type: string
-
+      version:
+        description: 'The version to release (e.g. 1.2.3). This workflow will automatically perform the required version bumps'
+        required: true
       dry_run:
         description: If set, run a dry-run release
         default: false
         type: boolean
 
+env:
+  JAVA_VERSION: 17
+  JAVA_DIST: temurin
+  RELEASE_VERSION: ${{ inputs.version }}
```

Comment:
I wonder whether this could be shared between both workflows so we don't need to maintain these when a new Java version is needed. What do you think?

Reply:
I added a global maven config file for those maven arguments.
Ref: https://github.com/elastic/ecs-logging-java/pull/226/commits/1a83544c3ae6d381292790bf5b69dc34585b8ac7

Reply:
I also create a composite action to centralize maven install and execution
Ref: https://github.com/elastic/ecs-logging-java/pull/226/commits/aab1f0a927f5b9eb31c2c2924e831456bfe098d5